_data/mentors.yml

```diff
@@ -52,7 +52,7 @@
   github: jigyasa-grover
   image: jigyasa-grover.jpg
   facebook: grover.jigyasa
-  twitter: jigyasa_grover
+-   twitter: jigyasa_grover
```

Comment:
please don't include -, jekyll will parse it as a new mentor

Reply:
How do I change this? Do I need to make a new PR?

Reply:
Make the required changes.
Then, amend the commit and force push the branch.

```
git add _data/mentors.yml
git commit --amend
git push origin <branch-name> -f
```